client/.idea/misc.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
ovo se ne komituje, izbaci iz komita-a, to ces morati definisati ja mislim u iznad .gitignore da ti ne bi se komitovalo, znaci u folderu gdje se nalaze i `server` i `client`

Reply:
Da kreiram .gitignore u folderu iznad, je l to?